internal/core/timers.rs

```diff
@@ -182,7 +182,8 @@ impl Drop for Timer {
     fn drop(&mut self) {
         if let Some(id) = self.id() {
             let _ = CURRENT_TIMERS.try_with(|timers| {
-                timers.borrow_mut().remove_timer(id);
+                let callback = timers.borrow_mut().remove_timer(id);
+                drop(callback);
```

Comment:
```suggestion
                let callback = timers.borrow_mut().remove_timer(id);
                // drop the callback without having CURRENT_TIMERS borrowed
                drop(callback);
```

I was wondering why you would write it like that in two lines. I think a comment can be helpful.

Reply:
Good idea